src/main/kotlin/InputView.kt

```diff
@@ -0,0 +1,32 @@
+import domain.MoveStrategy
+import java.util.*
+
+private val scanner = Scanner(System.`in`)
+
+fun getUserInputNames(): List<String> {
+    println(
+        """
+        안녕하세요, 자동차 경주 게임에 오신걸 환영합니다.
+        사람의 이름으로 자동차 이름이 결정됩니다. 
+        함께 하고자 하는 사람들의 이름을 , 를 기준으로 입력해주세요.
+    """.trimIndent()
+    )
+
+    val scanner = Scanner(System.`in`)
+
+    val userInputNames = scanner.nextLine()
+    return userInputNames.split(",").toList()
+}
+
+fun getUserInputMoveStrategy(): String {
+    println("자동차가 움직이는 전략을 결정할 수 있습니다. 전략과 설명은 아래와 같습니다.")
+    MoveStrategy.values()
+        .forEach { value -> println("""전략 이름 : ${value.name} 전략 설명 : ${value.description}""") }
```

Comment:
```suggestion
        .forEach { println("""전략 이름 : ${it.name} 전략 설명 : ${it.description}""") }
```
크게 구분이 어렵지 않거나 중첩된 람다가 아니라면 보통은 `it`으로 사용하는 편입니다. 익숙해지시면 보기 괜찮을거에요

Reply:
코틀린은 stream이 없어서, 줄바꿈 단위가 애매한데 보통 어떻게 줄바꿈하나요?

Reply:
```kotlin
.forEach {
  ~~~
}.forEach {
  ~~~
}


.forEach { ~~ }
.forEach { ~~ }
```
크게 두가지가 있는거 같은데 전자는 인텔리제이가 `it`이 어떤 타입인지 알려주고 후자는 알려주지 않아요
근데 생긴건 후자가 더 이뻐서 그냥 그때그때 섞어씁니다.